### PR TITLE
Add string and double array scale attributes

### PR DESCRIPTION
**Pull request by astarche**
### What does this Pull Request accomplish?

Add DAQ Scale string and double array attributes.
Regenerating PowerUpStates metadata with automated process.

### Why should this Pull Request be merged?

Expands support and testing for attribute data types.
Completes Get/Set Scale attributes.

### What testing has been done?

Ran and passed all DAQ tests.

### CODE REVIEW

File: source/codegen/metadata/nidaqmx/enums.py [hunk 1]
```diff
@@ -1156,19 +1156,17 @@
     'PowerUpStates': {
         'values': [
             {
-                'documentation': {
```
Comment (astarche) [inline source/codegen/metadata/nidaqmx/enums.py:1159]: Mentioned in the PR description, I'm not generating these so they look a little different but the output is the same (Except for the extra enum option).

File: source/tests/system/nidaqmx_driver_api_tests.cpp [hunk 2]
```diff
@@ -914,14 +974,68 @@ TEST_F(NiDAQmxDriverApiTests, SetPreScaledUnits_GetPreScaledUnits_ReturnsAttribu
...
+  EXPECT_EQ(UNITS, response.value());
+}
+
+
+TEST_F(NiDAQmxDriverApiTests, SetPolynomialForwardCoefficients_GetPolynomialForwardCoefficients_ReturnsAttribute)
+{
+  const auto SCALE_NAME = std::string("TestPolynomialScale");
+  const auto INITIAL_COEFFICIENTS = std::vector<double>{1.0, 2.0, 3.0};
+  const auto COEFFICIENTS = std::vector<double>{1.0, 3.0, 8.0};
+  auto scale_status = create_polynomial_scale(SCALE_NAME, INITIAL_COEFFICIENTS, INITIAL_COEFFICIENTS);
```
Comment (reckenro) [inline source/tests/system/nidaqmx_driver_api_tests.cpp:1018]: Just to make sure I understand, this call for `create_polynomial_scale` is to set the forward and reverse coefficients so later in validation we're making sure we're not getting either of those initial coefficients, right?

**Review by reckenro — APPROVED** [review]

File: source/tests/system/nidaqmx_driver_api_tests.cpp (hunk 2 above)
Comment (astarche) [reply source/tests/system/nidaqmx_driver_api_tests.cpp:1018]: Yeah, I'd sort of prefer not to set the initial value at all. But the create method requires that you pass something in. So we initialize to some value. Then assert that we can write-and-read-back some other value.

**Comment by astarche** [comment]
Add DAQ Scale string and double array attributes.
Regenerating PowerUpStates metadata with automated process.

**Pull request merged**